src/components/flow-item/flow-item.tsx

```diff
@@ -0,0 +1,271 @@
+   */
+  @Method()
+  async setFocus(): Promise<void> {
+    const { backButtonEl, containerEl } = this;
+
+    if (backButtonEl) {
+      backButtonEl.setFocus();
+      return;
+    }
+
+    containerEl?.setFocus();
+  }
+
+  /**
+   * Scrolls the component's content to a specified set of coordinates.
+   *
+   * ```
+   *   myCalciteFlowItem.scrollContentTo({
+   *     left: 0, // Specifies the number of pixels along the X axis to scroll the window or element.
+   *     top: 0, // Specifies the number of pixels along the Y axis to scroll the window or element
+   *     behavior: "auto" // Specifies whether the scrolling should animate smoothly (smooth), or happen instantly in a single jump (auto, the default value).
+   *   });
+   * ```
+   *
+   * @param options
+   */
+  @Method()
+  async scrollContentTo(options?: ScrollToOptions): Promise<void> {
+    this.containerEl?.scrollContentTo(options);
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  backButtonClick = (): void => {
+    this.calciteFlowItemBackClick.emit();
+  };
+
+  setBackRef = (node: HTMLCalciteActionElement): void => {
+    this.backButtonEl = node;
+  };
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+
+  renderBackButton(): VNode {
+    const { el } = this;
+
+    const rtl = getElementDir(el) === "rtl";
+    const { showBackButton, intlBack, backButtonClick } = this;
+    const label = intlBack || TEXT.back;
+    const icon = rtl ? ICONS.backRight : ICONS.backLeft;
+
+    return showBackButton ? (
+      <calcite-action
+        aria-label={label}
+        class={CSS.backButton}
+        icon={icon}
+        key="flow-back-button"
+        onClick={backButtonClick}
+        ref={this.setBackRef}
+        scale="s"
+        slot="header-actions-start"
+        text={label}
+      />
+    ) : null;
+  }
+
+  render(): VNode {
+    const {
+      closable,
+      closed,
+      description,
+      disabled,
+      heading,
+      headingLevel,
+      heightScale,
+      intlBack,
+      intlClose,
+      intlOptions,
+      loading,
+      menuOpen,
+      widthScale
+    } = this;
+    return (
+      <Host>
+        <calcite-panel
+          closable={closable}
+          closed={closed}
+          description={description}
+          disabled={disabled}
+          heading={heading}
+          headingLevel={headingLevel}
+          heightScale={heightScale}
+          intlBack={intlBack}
+          intlClose={intlClose}
+          intlOptions={intlOptions}
+          loading={loading}
+          menuOpen={menuOpen}
+          widthScale={widthScale}
+        >
+          <slot name={SLOTS.headerActionsStart} slot={PANEL_SLOTS.headerActionsStart} />
```

Comment:
@jcfranco can you revert the changes you made using `getSlotted()`? That won't work with this because getSlotted will always return true in this case.

We should not use getSlotted anymore because it won't work if nested slots exist.

Reply:
Sorry, I unintentionally rolled back changes to panel. Can you take another look?

Reply:
Changes look good now! ✨